azurerm/internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -177,6 +177,17 @@ func resourceLinuxVirtualMachine() *schema.Resource {
 
 			"identity": virtualMachineIdentitySchema(),
 
+			"license_type": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Computed:         true,
+				DiffSuppressFunc: suppress.CaseDifference,
+				ValidateFunc: validation.StringInSlice([]string{
+					"RHEL_BYOS",
+					"SLES_BYOS",
+				}, true),
```

Comment:
can we make this case-sensitive?

```suggestion
				}, false),
```